doc/users/next_whats_new/steppatch_and_stairs.rst

```diff
@@ -0,0 +1,22 @@
+New `~.matplotlib.patches.StepPatch` artist and a `.pyplot.stairs` method
```

Comment:
Please avoid linking in the title, as it does weird things in the heading and table of contents. Links should go in the text.